src/backend/distributed/commands/table.c

```diff
@@ -1469,6 +1621,26 @@ PostprocessAlterTableStmt(AlterTableStmt *alterTableStatement)
 			}
 		}
 	}
+
+	/* for the new sequences coming with this ALTER TABLE statement */
+	if (ShouldSyncTableMetadata(relationId))
+	{
+		List *sequenceCommandList = NIL;
+
+		/* commands to create sequences */
+		List *sequenceDDLCommands = SequenceDDLCommandsForTable(relationId);
+		sequenceCommandList = list_concat(sequenceCommandList, sequenceDDLCommands);
+
+		/* prevent recursive propagation */
+		SendCommandToWorkersWithMetadata(DISABLE_DDL_PROPAGATION);
```

Comment:
we usually need to re-enable it. 

It could also be nice to add it to the list.

Reply:
ping